tests/framework/clients/object.go

```diff
@@ -41,9 +41,12 @@ func CreateObjectOperation(k8sh *utils.K8sHelper, manifests installer.CephManife
 
 // ObjectCreate Function to create a object store in rook
 func (o *ObjectOperation) Create(namespace, storeName string, replicaCount int32) error {
-
+	securePort := " 443"
```

Comment:
```suggestion
	securePort := "443"
```

Reply:
That was actually intentional, so that we wouldn't have a trailing space hanging out in the empty securePort case. Still, looks like this method is going entirely, so that's no longer relevant.